### PR TITLE
Fixes and improvements

### PR DESCRIPTION
**Pull request by s0undt3ch**

### CODE REVIEW

File: scripts/bootstrap-salt-minion.sh [hunk 1]
```diff
@@ -438,18 +451,21 @@ install_ubuntu_daily() {
...
 }
 
 install_ubuntu_git_post() {
-    service salt-minion start
+    for fname in $(echo "minion master syndic"); do
+        if [ $fname != "minion" ]; then
+            # Guess we should only enable and start the minion service. Right??
+            continue
+        fi
+        cp ${SALT_GIT_CHECKOUT_DIR}/debian/salt-$fname.init /etc/init.d/salt-$fname
```
Comment (akoumjian) [inline scripts/bootstrap-salt-minion.sh:464]: Doesn't L455 already setup the service init scripts when using --install-layout=deb? If not, that's fine.

**Comment by akoumjian** [comment]
Should we be adding a flag that indicates whether or not the user wants salt-minion to start running? As usual, great stuff here!

File: scripts/bootstrap-salt-minion.sh (hunk 1 above)
Comment (s0undt3ch) [inline scripts/bootstrap-salt-minion.sh:464]: I don't think so, hence me adding it. Want me to first test if the init script exists before overriding it?

**Comment by s0undt3ch** [comment]
About the start flag, that might be a good idea. Debian and Ubuntu start services by default, Fedora does not, with our flag, we'll start(or at least try) the service.

File: scripts/bootstrap-salt-minion.sh (hunk 1 above)
Comment (akoumjian) [inline scripts/bootstrap-salt-minion.sh:464]: Nah, like you said you probably added it because it wasn't happening.

**Comment by akoumjian** [comment]
I forgot about debian's default with starting services.

**Comment by s0undt3ch** [comment]
Still about the start flag, this script is supposed to bootstrap salt, so, it's supposed to start salt, right?

**Comment by akoumjian** [comment]
Well, at least install. But probably in most cases start it as well.

**Pull request merged**